api/v1alpha1/tls_types.go

```diff
@@ -5,8 +5,8 @@
 
 package v1alpha1
 
-// +kubebuilder:validation:XValidation:rule="has(self.minVersion) && self.minVersion == 'v1_3' ? !has(self.ciphers) : true", message="setting ciphers has no effect if the minimum possible TLS version is 1.3"
```

Comment:
can you add some test for these in a separated PR.

Reply:
Added #2424